web/profiles/dpl_cms/translations/da.po

```diff
@@ -2236,6 +2236,13 @@ msgctxt "Favorites list"
 msgid "@count materials"
 msgstr "@count materialer"
 
+msgctxt "dpl_event"
+msgid "Access"
+msgstr ""
```

Comment:
Why do you translate it to an empty string?
(This is happening several times in this file)